test.js

```diff
@@ -10,10 +10,9 @@ function deserializeNonError(t, value) {
 
 test('main', t => {
 	const serialized = serializeError(new Error('foo'));
-	const keys = Object.keys(serialized);
-	t.true(keys.includes('name'));
-	t.true(keys.includes('stack'));
-	t.true(keys.includes('message'));
+	t.truthy(serialized.name);
+	t.truthy(serialized.stack);
+	t.truthy(serialized.message);
```

Comment:
Not sure if this is good enough

Reply:
I would do `t.true('message' in serialized);`